ui/app/components/secure-variable-form.js

```diff
@@ -137,6 +149,15 @@ export default class SecureVariableFormComponent extends Component {
         this.keyValues = nonEmptyItems;
       }
 
+      if (this.args.model?.isNew) {
+        if (this.namespaceOptions) {
+          this.args.model.set('namespace', this.variableNamespace);
+        } else {
+          const [namespace] = this.store.peekAll('namespace');
```

Comment:
I don't think this works; `store` is not defined in this file / injected as a service, and I get the following on save:

![image](https://user-images.githubusercontent.com/713991/178825103-71766954-2253-4e5d-8e3d-f79c67714638.png)

Reply:
We were missing store injection. Should be resolved now.